packages/pilot/public/locales/pt/translations.json

```
@@ -464,6 +464,8 @@
       "automatic_by_volume": "Automático por volume",
       "automatic_by_volume_explanation_1": "Antecipe parte das suas vendas. Nesse caso, você recebe diariamente um percentual do volume financeiro que tem a receber. ATENÇÃO: é preciso levar em conta que o percentual liberado para antecipar foi pré-fixado nas suas negociações, e este percentual varia de acordo com o volume que você já está antecipando. Para mais informações, veja:",
       "automatic_by_volume_explanation_link": " Qual percentual das vendas pode ser recebido antecipadamente?",
+      "compulsory": "Antecipação compulsória",
+      "compulsory_explanation_1": "Na antecipação compulsória todas as suas vendas já nascem com uma data fixa para serem pagas. Por exemplo, se você contratou a pagar.me para receber suas compras em 15 dias, todas as suas vendas já nascem antecipadas para serem pagas daqui a 15 dias.",
```

@victorteodoro pq o nome do campo tem `1` no final? :thinking:

```suggestion
      "compulsory_explanation_1": "Na antecipação compulsória todas as suas vendas já nascem com uma data fixa para serem pagas. Por exemplo, se você contratou o Pagar.me para receber suas compras em 15 dias, todas as suas vendas já nascem antecipadas para serem pagas daqui a 15 dias.",
```

---

@leonampd é pra respeitar o formato das outras strings de explicação. Se vocês der uma olhada todas as strings seguem esse formato :point_up: